### ISSUE TITLE
Why are the --silent and --logging-program arguments mutally exclusive?

### ISSUE THREAD

**Issue by Djelibeybi** [delfick/delfick_project#1]
I'm a bit confused about the logic behind making `--silent` and `--logging-program` mutally exclusive, specifically when using `--syslog-address /dev/log`. 

In this case, I don't want any of the `lifx` script's info messages, I just want any errors that may occur to be sent to the syslog, with the right program name so I can find them later. 

What am I missing?

**Comment by delfick** [comment]
I'll have time to run it later and see but afaict from a quick look, they aren't mutually exclusive

**Comment by Djelibeybi** [comment]
Tell that to the `lifx` script:

```
$ lifx --silent --logging-program lifx --syslog-address /dev/log --task lan:transform --reference d073d5xxxxxx --  '{ "power": "off" }'
usage: lifx [-h]
            [--verbose | --silent | --debug | --logging-program LOGGING_PROGRAM]
            [--tcp-logging-address TCP_LOGGING_ADDRESS]
            [--udp-logging-address UDP_LOGGING_ADDRESS]
            [--syslog-address SYSLOG_ADDRESS] [--json-console-logs]
            [--version] [--logging-handler-file LOGGING_HANDLER_FILE]
            [--dry-run] [--task PHOTONS_APP_CHOSEN_TASK]
            [--artifact PHOTONS_APP_ARTIFACT]
            [--reference PHOTONS_APP_REFERENCE] [--config PHOTONS_APP_CONFIG]
lifx: error: argument --logging-program: not allowed with argument --silent
```

**Comment by delfick** [comment]
ah, yeap,I read it wrong

change https://github.com/delfick/delfick_project/blob/master/delfick_project/app.py#L561 to parser instead of logging.add_argument.

And see if that works.

(in your virtualenv do a `pip uninstall delfick_project` and then from your checkout of this repo with that change, run `pip install -e .` in your virtualenv and see if that fixes it, and then feel free to make a PR)

**Comment by delfick** [comment]
Ok, I have released photons-core==0.24.1 with this change :)
